cScripts/CavFnc/functions/modules/fn_moduleCallEndex.sqf

```diff
@@ -11,15 +13,73 @@
  *
  * Public: No
  */
+ 
+//Module Options
+private _dialogResults = [
+	"Endex Options",
+	[
+		["Set Player Weapons To Safe", ["Yes", "No"], 0],
+		["Pacify AI", ["Yes", "No"], 0],
+		["Hint Players That Fire", ["Yes", "No"], 0]
+	]
+] call Ares_fnc_showChooseDialog;
 
-"ENDEX ENDEX ENDEX" remoteExecCall ["systemChat", 0];
-"DEBRIEF IN GAME" remoteExecCall ["systemChat", 0];
-"LEADERS DEBRIEF IN CHANNEL BELOW" remoteExecCall ["systemChat", 0];
+if (count _dialogResults == 0) exitWith {};
 
+//handles results and turns them into booleans
+_dialogResults params ["_weaponsSafe", "_aiPacified", "_VCSTIME"];
```

Comment:
Your solutions sexier seting the `params` instead of just the `select` one I like it!

Tho rename the `VCSTIME` fun and true yes but use something more understandable :P `_holdFireMessage` or somthing